openage/convert/hardcoded/texture.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2016-2016 the openage authors. See copying.md for legal info.
+
+"""
+Constants for texture generation.
+"""
+
+MAX_TEXTURE_DIMENSION = 4096
+"""
+The maximum texture dimension that can be expected to be supported by all GPUs.
+"""
+
+MARGIN = 1
+"""
+A margin between subtextures in an atlas to avoid texture bleeding.
+"""
```

Comment:
These are called attribute docstrings by [PEP 257](https://www.python.org/dev/peps/pep-0257/). While there is no direct interpreter support, tools like Sphinx know them. That said - of course I'll change it for consistency if you like.